### ISSUE TITLE
Questions about pre-training

### ISSUE THREAD

**Issue by karchen-geng** [workforai/QDMN#11]
Hello, if you only use static_ Can images (stage-0) and daivs-youtube (stage-2) be trained?
How much worse is the accuracy of the model than that of the model containing BL30K?

**Comment by workforai** [comment]
Yes, the default training setting is s02. The corresponding results can be seen in our paper.

**Comment by karchen-geng** [comment]
> Yes, the default training setting is s02. The corresponding results can be seen in our paper.

thanks

**Comment by karchen-geng** [comment]
请问您知道这个模型在windows系统上如何运行吗，同样的运行方式，总是报numpy的问题

**Comment by karchen-geng** [comment]
> 请问您知道这个模型在windows系统上如何运行吗，同样的运行方式，总是报numpy的问题

No module named 'numpy.core._multiarray_tests'

**Comment by workforai** [comment]
Sorry, I have not tried to run it on Windows, so I can't assist you.

**Comment by karchen-geng** [comment]
您好，我只用stage-0和stage-2进行预训练，模型得出的结果在DAVIS2017_val中测试出的结果与用您的s012模型相差 8个点，请问这样正常吗嘛？

**Comment by workforai** [comment]
不正常，是完全一样的训练setting，没有改动任何代码吗？

**Comment by karchen-geng** [comment]
对，只是改了数据集的位置和batchsize的大小，youtube用的是2019的，不是all_frame

**Comment by workforai** [comment]
你改了batchsize大小以后调iteration了吗

**Comment by karchen-geng** [comment]
没有

**Comment by workforai** [comment]
那你没训够啊，你是不是把batchsize减小了？要保证训练的epoch一样啊

**Comment by karchen-geng** [comment]
等于ieration 与 batchsize的乘积要保持一致？是这样吗？

**Comment by workforai** [comment]
iteration与所有GPU加起来总的batch的乘积保持一致

**Comment by karchen-geng** [comment]
我看您的是两个GPU，我的是单个GPU，那我的还要乘以2？

**Comment by workforai** [comment]
Yes

**Comment by karchen-geng** [comment]
十分感谢！！！

**Comment by workforai** [comment]
而且如果总batchsize不一致，即使epoch一样，结果可能也会有一定的偏差的

**Comment by karchen-geng** [comment]
好的好的，我再调一调

**Comment by karchen-geng** [comment]
train_sampler, train_loader = renew_loader(cur_skip, cur_skip_davis)
您好，请问每过几个epoch , 进行跳帧这个操作的意图是什么？
